connectors/sink-tencentcloud-scf/internal/server.go

```diff
@@ -65,56 +71,56 @@ func NewFunctionSink() connector.Sink {
 	return &functionSink{}
 }
 
+var _ connector.Sink = &functionSink{}
+
 type functionSink struct {
 	scfClient *v20180416.Client
-	logger    log.Logger
 	cfg       *Config
 	funcName  string
 }
 
-func (c *functionSink) Receive(_ context.Context, event v2.Event) protocol.Result {
+func (c *functionSink) Arrived(_ context.Context, events ...*v2.Event) connector.Result {
 	req := v20180416.NewInvokeRequest()
 	req.FunctionName = &c.cfg.F.Name
 	req.Namespace = &c.cfg.F.Namespace
-	payload := string(event.Data())
-	req.ClientContext = &payload
 
-	res, err := c.scfClient.Invoke(req)
-	if err != nil {
-		c.logger.Debug("failed to invoke function", map[string]interface{}{
-			log.KeyError: err,
+	for idx := range events {
+		e := events[idx]
+		payload := string(e.Data())
+		req.ClientContext = &payload
+
+		res, err := c.scfClient.Invoke(req)
+		if err != nil {
+			log.Debug("failed to invoke function", map[string]interface{}{
+				log.KeyError: err,
+			})
+			return connector.NewResult(http.StatusInternalServerError, err.Error())
+		}
+		log.Debug("invoke function success", map[string]interface{}{
+			"response": res,
 		})
-		return v2.NewHTTPResult(http.StatusInternalServerError, err.Error())
 	}
-	c.logger.Debug("invoke function success", map[string]interface{}{
-		"response": res,
-	})
-	return protocol.ResultACK
-}
 
-func (c *functionSink) Init(cfgPath, secretPath string) error {
-	cfg := &Config{}
-	if err := cdkutil.ParseConfig(cfgPath, cfg); err != nil {
-		return err
-	}
+	return connector.Success
```

Comment:
```suggestion
	return cdkgo.Success
```

Reply:
`Success` no in aliases

Reply:
I forget it, it is `SuccessResult` not `Success`